addons/website/static/src/builder/plugins/customize_website_plugin.js

```diff
@@ -715,7 +715,9 @@ export class CustomizeWebsiteVariableAction extends BuilderAction {
     }
     isApplied({ params: { mainParam: variable } = {}, value }) {
         const currentValue = this.dependencies.customizeWebsite.getWebsiteVariableValue(variable);
-        return currentValue === value;
+        return (
+            currentValue === value || `'${currentValue}'` === value || currentValue === `'${value}'`
+        );
```

Comment:
Why do you want to support multi version ? Or at least for the value ?

Reply:
Ok, we only need to support it on current value - given that it has now been used on actual databases without this fix.